apps/k8s-io/configmap-nginx.yaml

```diff
@@ -54,6 +54,11 @@ data:
             ';
           }
 
+          # Redirect go import paths to Go package documentation.
+          if ($repo ~ "api|apimachinery|client-go|kubernetes") {
+            return 301 https://pkg.go.dev/k8s.io/$repo/$subpath;
```

Comment:
I'd use a 302 redirect. 301 implies either “this is the new home for the content that used to be here, and we're sure of that” or “having consulted a soothsayer, we know we will never redirect anywhere else” (and I think the second meaning is ambitious).

Reply:
Excellent call, done.